tools/cibadmin.c

```diff
@@ -120,6 +124,29 @@ static pcmk__cli_option_t long_options[] = {
         "md5-sum-versioned", no_argument, NULL, '6',
         "Calculate an on-the-wire versioned CIB digest", pcmk__option_default
     },
+#if ENABLE_ACL
```

Comment:
You can drop all references to ENABLE_ACL -- ACLs are now always supported, and the constant is no longer defined

Reply:
Since we moved pcmk__acl_evaled_as_namespaces to pcmki, there is a linker error. What's the best way to fix it?

cibadmin.o: In function `main':
/home/gchin/pacemaker/tools/cibadmin.c:790: undefined reference to `pcmk__acl_evaled_as_namespaces'
/home/gchin/pacemaker/tools/cibadmin.c:808: undefined reference to `pcmk__acl_evaled_render'

Reply:
It's been fixed